wpilibj/src/main/java/edu/wpi/first/wpilibj/DriverStation.java

```diff
@@ -189,6 +197,89 @@ private static void reportErrorImpl(boolean isError, int code, String error, boo
   }
 
   /**
+   * The state of one joystick button. Button indexes begin at 1.
+   *
+   * @param stick  The joystick to read.
+   * @param button The button index, beginning at 1.
+   * @return The state of the joystick button.
+   */
+  public boolean getStickButton(final int stick, byte button) {
```

Comment:
Isn't byte kind of annoying to use?  Why not int?

Reply:
Because it has always been that type. I just moved the function definition to a different location in the file. The types _are_ currently inconsistent though, so I'll change it.